src/output-text.js

```diff
@@ -5,12 +5,13 @@ import ErrorCodes from "./error-codes.js";
 import defaultArgs from "./default-args.js";
 
 import type {ErrorCode} from "./error-codes.js";
-import type {ILog, Options} from "./types";
+import type {ILog, Options, JsonItem} from "./types";
 
 const outputText = (
     options: Options,
-    violationFileNames: Array<string>,
     log: ILog,
+    jsonItems: Array<JsonItem>,
+    violationFileNames: Array<string>,
```

Comment:
I like that you've moved the more optional args to the end.

Reply:
Hopefully we can merge those last two in the future.  Having them both at the end should make that a bit easier.